tests/AssertTraitTest.php

```diff
@@ -94,7 +84,7 @@ public function testAssertJsonMatchesSchemaString()
      */
     public function testAssertJsonValueEquals($expression, $value)
     {
-        $content = json_decode(file_get_contents(self::getJson('testAssertJsonValueEquals.json')));
+        $content = json_decode(file_get_contents(Utils::getJsonPath('testAssertJsonValueEquals.json')));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class 'EnricoStahn\JsonAssert\Tests\Utils' in method 'testAssertJsonValueEquals'.](https://www.codacy.com/app/estahn/phpunit-json-assertions/file/2256991340/issues/source?bid=3181605&fileBranchId=3182397#l87)